setup/lint.py

```diff
@@ -0,0 +1,12 @@
+"""Modules"""
```

Comment:
The docstring does not explain what this module does. Please add more detail.
The first line of the docstring does not end in a period (full stop), Please run pydocstring to verify.